tests/test_examples.py

```diff
@@ -220,6 +220,8 @@ def to_test(model_name: str, multi_card: bool, deepspeed: bool, example_name: st
             return True
         elif "bloom" in model_name and deepspeed and os.environ.get("GAUDI2_CI", "0") == "0":
             return True
+        elif "ast-finetuned-speech-commands-v2" in model_name and os.environ.get("GAUDI2_CI", "0") == "1":
+            return True
```

Comment:
You need to move this right before line 195, otherwise this test will also be executed on Gaudi1